domino/lib/src/experimental/idom.dart

```diff
@@ -10,6 +10,8 @@ abstract class DomContext<L, V> {
   L get element;
   dynamic get pointer;
 
+  final globals = {};
```

Comment:
Let's use a single, read-only `locale` field for now (as there is no other use for `globals` yet). Or rather... create a ` class DomContextGlobals{ final String locale;}` and reference it that as typed globals.